splitter.d

```diff
@@ -488,11 +488,11 @@ struct DSplitter
 					goto default;
 				break;
 			case '@':
-				if (consume("disable")
-				 || consume("property")
-				 || consume("safe")
-				 || consume("trusted")
-				 || consume("system")
+				if (consume("disable") !is null
+				 || consume("property") !is null
+				 || consume("safe") !is null
+				 || consume("trusted") !is null
+				 || consume("system") !is null
```

Comment:
Thank you for your comment.

I changed `DSplitter.consume(string)` to return `bool`.
I also changed `DSplitter.consume(char)` to return `bool`.
They make consistent with a local function `bool consume(Token)` which is defined in `DSplitter.postProcessBlockStatements`.